src/pytest_cov/plugin.py

```diff
@@ -50,6 +50,12 @@ def __call__(self, parser, namespace, values, option_string=None):
 def pytest_addoption(parser):
     """Add options to control coverage."""
 
+    def fail_under(num_str):
```

Comment:
Can we have this at module level, like the other argument type, perhaps as `parse_fail_under`? It doesn't need to be inside this closure.